packages/zowe-explorer/src/job/ZosJobsProvider.ts

```diff
@@ -84,17 +84,18 @@ export class ZosJobsProvider extends ZoweTreeProvider implements IZoweTree<IZowe
         {
             key: `owner`,
             label: `Job Owner`,
-            value: undefined,
+            value: "",
```

Comment:
I'm not opposed to this change, but is this the root cause of the discussion regarding `!=`? 😋

Reply:
There are a few assignments in the `handleEditingMultiJobParameters` function that use this `.value` property - e.g. `jobProperties.find((prop) => prop.key === "owner").value`. This just ensures that `.value` will never be undefined for the owner value (the name of the job owner). Although they are both falsy values this will prevent setting one of the node properties, e.g. `node.owner` to something that isn't a string (undefined).

Reply:
yeah, that makes sense 😋 
I thought that this `value: undefined` was one of the possible reasons 😋 
Thanks for explaining though 😅